en/editor-guidelines.md

```diff
@@ -197,7 +197,7 @@ Once you and the author are happy with a tutorial, the next step is to begin the
 
 ### 1) Create an Author Bio
 
-If the lesson has been written by a new author, editors should add information about the author to the site's [authors directory](https://github.com/programminghistorian/jekyll/blob/gh-pages/_data/ph_authors.yml). Follow the syntax for the examples already included there:
+If the lesson has been written by a new author, the managing editor will need to add a new bio for that person. You will need to provide the managing editor with a version of the following information:
 
 ```yaml
 - name: Jim Clifford
```

Comment:
this section includes information on the lesson but no space for adding the actual bio